test/README.md

```diff
@@ -1,5 +1,63 @@
+   * The CDK context options for this CDK can be used to customize this deployment and can be provided with a custom context file using the `--context-file` option
+   * A sample CDK context file can be seen [here](defaultCDKContext.json)
+2. Build the Docker Images
+3. Deploy the Migration Assistant CDK (containing the Migration tooling and potentially an OpenSearch Domain) located in this repo [here](../deployment/cdk/opensearch-service-migration)
+   * The CDK context options for this CDK can be used to customize this deployment and can be provided with a custom context file using the `--context-file` option
+   * A sample CDK context file can be seen [here](defaultCDKContext.json)
+4. Add the traffic stream source security group created by the Migration Assistant CDK to the source cluster load balancer
+5. Run the `./startCaptureProxy.sh` script for starting the Capture Proxy process on each source cluster node/instance 
+   * This script is automatically added to each EC2 instance on creation
+
+#### Configuring a Historical Migration focused E2E environment
+A more focused experience for only testing historical migrations can be achieved with this script by customizing the CDK context options to minimize live capture resources like so:
+```
+{
+  "rfs-backfill": {
+    "stage": "<STAGE>",
+    "vpcId": "<VPC_ID>",
+    "engineVersion": "OS_2.11",
+    "domainName": "os-cluster-<STAGE>",
+    "dataNodeCount": 2,
+    "openAccessPolicyEnabled": true,
+    "domainRemovalPolicy": "DESTROY",
+    "artifactBucketRemovalPolicy": "DESTROY",
+    "kafkaBrokerServiceEnabled": true,
+    "trafficReplayerServiceEnabled": false,
+    "reindexFromSnapshotServiceEnabled": true,
+    "sourceClusterEndpoint": "<SOURCE_CLUSTER_ENDPOINT>"
+  },
+  "source-single-node-ec2": {
+    "suffix": "ec2-source-<STAGE>",
+    "networkStackSuffix": "ec2-source-<STAGE>",
+    "distVersion": "7.10.2",
+    "distributionUrl": "https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-oss-7.10.2-linux-x86_64.tar.gz",
+    "captureProxyEnabled": false,
+    "securityDisabled": true,
+    "minDistribution": false,
+    "cpuArch": "x64",
+    "isInternal": true,
+    "singleNodeCluster": true,
+    "networkAvailabilityZones": 2,
+    "dataNodeCount": 1,
+    "managerNodeCount": 0,
+    "serverAccessType": "ipv4",
+    "restrictServerAccessTo": "0.0.0.0/0"
+  }
+}
+```
+Note: That this deployment will still deploy an unused Kafka broker ECS service, as a traffic stream source is still required for Migration Assistant deployments, but this linkage should be untangled with: https://opensearch.atlassian.net/browse/MIGRATIONS-1532
+
+In the same vein, the `--skip-capture-proxy` option can be provided to skip any capture proxy setup on source cluster nodes
+```
+./awsE2ESolutionSetup.sh --context-file <CONTEXT_FILE> --migration-context-id rfs-backfill -skip-capture-proxy
+```
```

Comment:
This is probable exceeding the scope of the task you're currently working on, but - I'm curious why we have a command line parameter to skip the capture proxy setup instead of it just being an option via CDK Context?

Also - how do we specify the source cluster context we want to use in that file?

Reply:
The capture proxy setup we do currently is outside the scope of the CDK and is done via AWS CLI commands, which this script contains

There is another command argument to specify the id that I didn't include, but is probably clearer to include so have updated now